hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/ResponseSegment.java

```diff
@@ -3,17 +3,22 @@
 import java.util.Optional;
 
 import ca.bc.gov.hlth.hnsecure.parsing.Util;
+import ca.bc.gov.hlth.hnsecure.properties.ApplicationProperties;
+import ca.bc.gov.hlth.hnsecure.properties.ApplicationProperty;
 
 public abstract class ResponseSegment {
-	private static final String ack = "ACK";
 
-	private static final String fieldSeperator = "|";
+	protected static final String FIELD_SEPARATOR = "|";
```

Comment:
Could be removed and use Util.HL7_DELIMITER instead.

Reply:
Done.